compute/resource_job_test.go

```diff
@@ -452,3 +385,157 @@ func TestResourceJobDelete_Error(t *testing.T) {
 	qa.AssertErrorStartsWith(t, err, "Internal error happened")
 	assert.Equal(t, "789", d.Id())
 }
+
+func TestResourceJobCreateSingleNode(t *testing.T) {
```

Comment:
could you please also add `TestResourceJobCreateNewClusterFiveWorkers`? It was missing and it's important for this refactoring.

Reply:
nitpicking: could you move `TestResourceJobCreate*` test to be located right after creation tests? e.g. line 90+.